composer.json

```diff
@@ -48,5 +47,7 @@
         "branch-alias": {
             "dev-master": "2.1-dev"
         }
-    }
+    },
+    "minimum-stability":"dev",
```

Comment:
what other dev stability do we need? can't we do symfony-cmf/testing: ^2.1@dev instead of this line?

Reply:
is that same? i do not know.